pkgs/tools/security/masscan/default.nix

```
@@ -17,6 +17,11 @@ stdenv.mkDerivation rec {
     sha256 = "sha256-mnGC/moQANloR5ODwRjzJzBa55OEZ9QU+9WpAHxQE/g=";
   };
 
+  postPatch = lib.optionalString stdenv.isDarwin ''
```

```suggestion
  postPatch = ''
```
that shouldn't harm linux, right?

---

No, because the modified line would not be used on Linux:

https://github.com/robertdavidgraham/masscan/blob/06902c40d1494881c17b7968ef530e1d6bbd320c/Makefile#L41

But it is still a Darwin-only line, that's why I put the condition.

---

even better would be to upstream the fix, then fetch the patch.

---

nevermind, just read above comment

---

> that shouldn't harm linux, right?

I prefer the way it is. In case this package is updated and this line changes, this will only break Linux and not Darwin.